test/Renci.SshNet.IntegrationTests/SshTests.cs

```diff
@@ -63,7 +63,7 @@ public void Ssh_ShellStream_Exit()
                     catch (ObjectDisposedException ex)
                     {
                         Assert.IsNull(ex.InnerException);
-                        Assert.AreEqual("ShellStream", ex.ObjectName);
+                        Assert.AreEqual("Renci.SshNet.ShellStream", ex.ObjectName);
```

Comment:
is this unrelated?

Reply:
Yes, it was introduced in one of the recent analyzers fixups